src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -937,6 +937,16 @@ public IPointerTypeSymbol CreatePointerTypeSymbol(ITypeSymbol pointedAtType)
 
         protected abstract IPointerTypeSymbol CommonCreatePointerTypeSymbol(ITypeSymbol elementType);
 
+        /// <summary>
+        /// Returns a new INamedTypeSymbol representing a native integer.
```

Comment:
>Returns a new INamedTypeSymbol representing a native integer. [](start = 12, length = 61)

Consider reflecting the failure mode - an exception

Reply:
It looks like C# can return error types.

---
In reply to: [385417667](https://github.com/dotnet/roslyn/pull/41970#discussion_r385417667) [](ancestors = 385417667)

Reply:
Will add comment in subsequent PR that `VisualBasicCompilation` throws `NotSupportedException`.

---
In reply to: [385435975](https://github.com/dotnet/roslyn/pull/41970#discussion_r385435975) [](ancestors = 385435975,385417667)